tests/js/patching.js

```diff
@@ -13,9 +13,11 @@ suite('patching', function() {
   suite('HTMLElement', function () {
 
     test('constructor is configurable and writable', function() {
```

Comment:
This test doesn't really make sense. On [this line](https://github.com/webcomponents/custom-elements/blob/36451e1dbe4e3b77d98499fc12132573d91d4a8b/src/custom-elements.js#L679), the new `HTMLElement.prototype` is set to the old one, so the descriptor is exactly the same. So, I'm going to remove this.